plugins/lookup/passwordstore.py

```diff
@@ -154,6 +154,7 @@
 
 # backhacked check_output with input for python 2.7
 # http://stackoverflow.com/questions/10103551/passing-data-to-subprocess-check-output
+# note: contains special logic for calling 'pass', so not a drop-in replacement for check_output
```

Comment:
Ideally, this would be changed to a function called "run_pass" or similar and streamline everything below, but this seems too much for a fix that should be backported (and also takes more time).